bin/test/i18n/resources/pegasus/emails/test_sync_out.rb

```diff
@@ -0,0 +1,119 @@
+require_relative '../../../../test_helper'
+require_relative '../../../../../i18n/resources/pegasus/emails/sync_out'
+
+describe I18n::Resources::Pegasus::Emails::SyncOut do
+  def around
+    FakeFS.with_fresh {yield}
+  end
+
+  before do
+    STDOUT.stubs(:print)
+    I18n::Utils::PegasusMarkdown.stubs(:restore_file_header)
+  end
+
+  describe '.perform' do
+    it 'calls #process' do
+      I18n::Resources::Pegasus::Emails::SyncOut.any_instance.expects(:process).once
+
+      I18n::Resources::Pegasus::Emails::SyncOut.perform
+    end
+  end
+
+  describe '#process' do
+    let(:sync_out) {I18n::Resources::Pegasus::Emails::SyncOut.new}
+
+    let(:crowdin_locale) {'Not English'}
+    let(:i18n_locale) {'not-EN'}
+    let(:unique_language_code) {'expected_unique_language_code'}
+
+    let(:crowdin_locale_dir) {CDO.dir('i18n/locales', crowdin_locale)}
+    let(:crowdin_locale_resource_dir) {File.join(crowdin_locale_dir, 'pegasus/emails/public')}
```

Comment:
I believe public is not needed in the path here:
```suggestion
    let(:crowdin_locale_resource_dir) {File.join(crowdin_locale_dir, 'pegasus/emails')}
```